src/Pelagos/Bundle/AppBundle/Resources/public/css/dif.css

```diff
@@ -152,3 +152,16 @@ label:not(.button)
 {
     display:hidden;
 }
+
+fieldset {
+    margin-bottom: 1em;
+    padding: 0.5em;
+}
+
+#pelagos-content table {
+    border-collapse: collapse;
+}
+
+select, input{
```

Comment:
I would really like to see these standard elements prefixed with a class name.

Reply:
Yes, we should try to refactor in small chucks